### PR TITLE
Make single result page for images RTL-compatible

### LINKED ISSUE #435: Make single result page for images RTL compatible
Issues identified on this page:
- [x] Go to image's website" button needs to have its icon padding adjusted
- [x] The attribution box and tabs need to have their borders adjusted to be on the correct side for RTL languages
- [ ] The plain text attribution also needs to have its quotation marks corrected as currently they're being moved around by the browser into strange places
- [x] The license icon needs to have its padding adjusted against the associated text
- [x] The information table needs to have its padding adjusted as the "dimensions" text overlaps the listed dimensions
- [x] The leading edge of the information/reuse table needs to be fixed for RTL languages to align with the rest of the text in the result information

### PR DESCRIPTION
**Pull request by krysal**
## Fixes

Fixes #435 by @krysal

## Description

 This PR:
- Adjusts the padding of the "Go to Image's website"
- Adjusts the padding of license icons against the associated text on the "Reuse" tab
- Adjusts the padding of texts in the "Information" tab
- Deletes unused files, like `src/utils/get-provider-logo.js`, and old styles
- Removes the icon of the copy button
- Fixes the style of the image's title according to transitional mockups
- Adjusts overall padding and margin on the page
- ~The plain text attribution also needs to have its quotation marks corrected as currently they're being moved around by the browser into strange places~

## Testing Instructions

Set a right to left language (e.g. Arabic) using the locale chooser in the footer of a non-embedded page, like `http://localhost:8443?embedded=false` and check the fixed styles.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/CopyButton.vue [hunk 1]
```diff
@@ -6,11 +6,11 @@
     :data-clipboard-target="el"
   >
     <span v-if="!success">
-      <i class="icon cc-share me-2" />
+      <i class="icon cc-share mx-1" />
```
Comment (obulat) [inline src/components/CopyButton.vue:9]: The [transitional styles](https://www.figma.com/file/BlUTIXRHLOq5mRITpDpi3E/Transitional-styles?node-id=530%3A2051) don't have an icon on the copy button, so I think we can remove the icons altogether.

Comment (zackkrida) [reply src/components/CopyButton.vue:9]: Agreed, we should remove the icon. CC @panchovm

**Comment by zackkrida** [comment]
@krysal could you update this with some detail on what isn't complete? It's looking great!

File: src/components/CopyButton.vue (hunk 1 above)
Comment (krysal) [reply src/components/CopyButton.vue]: Good call, thanks. I should have checked the mockups before, It seemed strange to me that the title was so small, and of course, it's fixed in the transitional mockups :) I'll make that change too.

**Comment by krysal** [comment]
@zackkrida I added the info to the first post. The main thing that is left is to fix the quotation marks of the attribution text, I'll complete that today with some other minor styles fixes.

**Comment by krysal** [comment]
About the last point, I'm not sure if we can do anything here, as it seems correct for RTL languages that the opening quotation mark is at the right (beginning) of the text and the closing mark at the end of the word, but it's confusing with the text in English. It's the same case with periods on the index page.

However, I added the `dir="ltr"` to the textarea of the HTML, as I believe that would be always read from left to right.

**Review by obulat — CHANGES_REQUESTED** [review]
Looks great, and it's nice to see the code cleaned up more :) Can you also add the `dir="ltr"` to the HTML attribution span, as the quotes there are also placed incorrectly. This behavior is really weird, to be honest.

**Comment by krysal** [comment]
@obulat Could you clarify more specifically which `span` needs the `dir` attribute? I'm not sure which one you are referring to.

**Review by obulat — APPROVED** [review]
Since the rich text IS supposed to be translated, it doesn't need to be set to ltr. Great improvements overall 👌

**Review by zackkrida — APPROVED** [review]
The page looks beautiful in Firefox, Safari, and Chrome at all breakpoints with `lang=rtl`. Awesome work!

**Pull request merged**